src/Components/Blazor/WebAssembly.Authentication/src/Microsoft.AspNetCore.Components.WebAssembly.Authentication.csproj

```diff
@@ -0,0 +1,87 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <Sdk Name="Yarn.MSBuild" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <Description>Build client-side authentication for single-page applications (SPAs).</Description>
+    <IsShippingPackage>true</IsShippingPackage>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
+    <RazorLangVersion>3.0</RazorLangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Mono.WebAssembly.Interop" />
+    <Reference Include="Microsoft.AspNetCore.Blazor" />
+    <Reference Include="Microsoft.AspNetCore.Components.Web" />
+    <Reference Include="Microsoft.AspNetCore.Components.Authorization" />
+    <Reference Include="Microsoft.Extensions.Configuration" />
```

Comment:
I'm curious. Do we need all these references? I thought `Mono.WebAssembly.Interop` and `Microsoft.AspNetCore.Components.Web` were implied by `Microsoft.AspNetCore.Blazor`. Maybe I'm misremembering.

Reply:
Cleaned this up.